Sources/App/Settings/SettingsDetailViewController.swift

```diff
@@ -867,8 +878,8 @@ enum AppIcon: String, CaseIterable {
     case Purple = "purple"
     case Red = "red"
     case White = "white"
-    case BiPride = "bi_pride"
```

Comment:
what happens to users who had the old name set as their icon?

Reply:
Good catch, I have reverted the naming